### PR TITLE
double counting bug

### PR DESCRIPTION
**Pull request by as-polyakov**
I fixed the issue when certain paths would be double counted

### CODE REVIEW

File: new eva code.py [hunk 1]
```diff
@@ -28,6 +28,7 @@ def find(y,x,dy,dx,n,turn=False): #n is pointer (index)
     if n == 0:
         for i,m in hor+dia:
             find(y + i, x + m, i, m, n + 1, False)
+        return #QUIT here as we already stepped in all possible directions otherwise we will double step
```
Comment (as-polyakov) [inline new eva code.py]: I think this is a great idea!

Comment (Vtrifx) [inline new eva code.py]: hi gang u slayed that

**Review by Vtrifx — COMMENTED** [review]
hi gang u slayed that

File: new eva code.py (hunk 1 above)
Comment (as-polyakov) [reply new eva code.py]: thank you

Comment (Vtrifx) [reply new eva code.py]: we did NOT like the comment 🤓☝️

Comment (Vtrifx) [reply new eva code.py]: we did NOT like the comment, make it more simple 🤓☝️

Comment (as-polyakov) [reply new eva code.py]: done

**Pull request merged**